.github/workflows/all-modules-add-and-buildable.yaml

```diff
@@ -53,3 +59,14 @@ jobs:
       # Build
       - name: Status
         run: cfbs status
+      # Validate built policy
+      # Here we use root for everything to avoid permissions problems in policy that accesses files
+      - name: Validate built policy
+        run: |
+          sudo pip3 install cfbs
+          wget https://s3.amazonaws.com/cfengine.packages/quick-install-cfengine-enterprise.sh  && sudo bash ./quick-install-cfengine-enterprise.sh agent
+          sudo cfbs install
+          # fake a bootstrap, lib-fim module expects stdlib to be in inputs with $(sys.libdir)
+          sudo cp -r /var/cfengine/masterfiles/lib /var/cfengine/inputs
```

Comment:
How about installing and bootstrapping (for real) with cf-remote instead?

Reply:
well, I tried that, pretty sure this is in a container and networking isn't quite available, bootstrapping failed.

Reply:
@craigcomstock bootstrapping to localhost?

Reply:
Yes. `cf-agent -IB localhost` gave errors and didn't successfully bootstrap.

Reply:
Tried it again, captured the logs for us to fiddle with another time if we like.

Reply:
@craigcomstock alright, thanks for testing it :)